test/e2e/validation_test.go

```diff
@@ -21,30 +20,30 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kubeTypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
 )
 
 // waitForTrackerConfigMap to be created waits for the Windows tracker configmap to be created with appropriate values
-func waitForTrackerConfigMap(kubeclient kubernetes.Interface, namespace string, expectedNodesToBeTracked int,
-	retryInterval, timeout time.Duration) error {
+func (tc *testContext) waitForTrackerConfigMap() error {
```

Comment:
Please fix the doc given you are changing the function definition. There is an extra `to be created`